### ISSUE TITLE
Uninstaller removes scenes directory if using portable/working directory

### ISSUE THREAD

**Issue by jackjt8** [leMaik/chunky-launcher-standalone#4]
The uninstaller currently does not check for userdata present within the install location. If the user selects portable or working directory during the initial ChunkyLauncher run all data, including `\scenes\`, will be removed if the uninstaller is used. When using the default, `.chunky`, directory all data is safe.

Ideally, the uninstaller should have a list of files that should be removed and anything extra should not be touched. Informing the user of any leftover files would be the best approach.

**Comment by leMaik** [comment]
I disagree. User data should **never** be stored in program files and the user shouldn't assume that anything put there is persistent. We shouldn't offer the option to use that directory in the installer version and warn the user if they use it.

**Comment by jackjt8** [comment]
ChunkyLauncher's First-Time Setup defaults to the Home Directory which is typically on the C: drive. If we wish to store Chunky on another drive Portable or Working Directory are the ONLY options. We cannot migrate away from the a Home Directory .chunky install as there is no option to perform this in the Launcher or within Chunky. We also do not get a `chunky-launcher.json` to perform a manual migration. Typically the C: drive is a smaller SSD which we would not want to store a mass amount of data which cannot make full use out of the speed.

Furthermore, to make use of the Windows Package Manager we would need to not have a First-Time Setup.

I would suggest that we maybe use the documents library folder for the default location. As this folder can be relocated by Windows itself without any issues.
